### ISSUE TITLE
Source of questions?

### ISSUE THREAD

**Issue by Stephen-Gates** [mattwagstaff/ODICert_turnKey#4]
What was the source of questions? I can't see "Does your country have data protection laws?" in the GB FINAL Certificate.

**Comment by mattwagstaff** [comment]
Again, adding of a 'legend' will assist with this. 

Questions within the grey boxes are dependant on something/have a pre-requisite, therefore a question must be asked to ascertain whether these elements need to be included or not. If the answer is true, then include the elements in the grey box. 

I've adapted the questions from the text on the original "What will you be asked?" page here: https://certificates.theodi.org/overview.

A pure example of this is the API element, no certificate level *requires* you to publish an API per se, however *if you do* publish an API, you must include additional elements.

So, to use your example "Does your country have data protection laws?"... this has been adapted from the Legal section in the Pilot certificate where the text says:

"**...and if you have data protection laws like those in Europe**:
* publishing the privacy notice that individuals affected by the data consented to
* consulting with the person within your organisation who is responsible for data protection"

...does this ring true with you Steve?

**Comment by Stephen-Gates** [comment]
I don't believe https://certificates.theodi.org/overview represents the actual certificate questions. That's why I read the XML or try it online.

**Comment by mattwagstaff** [comment]
Ok, I'll read the XML and deduce from there. First pass was to try and give a better view of the requirements. It will refine as it progresses.
